api/managementpb/dbaas/psmdb_clusters.proto

```diff
@@ -129,6 +129,12 @@ message UpdatePSMDBClusterRequest {
   }
   // Cluster parameters.
   UpdatePSMDBClusterParams params = 3;
+  // Make DB cluster accessible outside of K8s cluster.
+  bool expose = 4;
+  // Make DB cluster accessible via public internet.
+  bool internet_facing = 5;
```

Comment:
was curious, is it possible for `internet_facing` to be true when `expose` is false? does that override the `expose` value and forces it to be internet facing?